### DISCUSSION TITLE
Insertion based on diff

### DISCUSSION THREAD

**Discussion by S1M0N38** [olimorris/codecompanion.nvim#145] [Ideas]
Any idea of implementing inline assistant insertion like zed ai?

I'm working on a new release of S1M0N38/dante.nvim (see dev branch). It use neovim built-in diff to highlight diff between generated code and original code. 

https://github.com/echasnovski/mini.diff could be a great addition for highlight diff inplace.

**Comment by olimorris** [comment]
This is great timing. I'm a few hours away from mentioning you in the README for your awesome work on dante.nvim. I've taken your approach to implement a diff viewer as per the conversation in #109. Oh and your footie odds Python app looks fantastic...I may check that out over the weekend.

Anyway, to the point of your post. I implemented "inline transformations" as per #130 a couple of days ago. How do they differ to Zed AI's? I'm struggling to keep track of all the new awesome developments we have in the world of LLMs each week.
